docs/docs/Concepts/mcp-server.mdx

```diff
@@ -186,31 +186,66 @@ For more information, see the MCP documentation for your client, such as [Cursor
 
 ### MCP server authentication and environment variables {#authentication}
 
-You must provide a Langflow API key in your MCP client configuration if your [Langflow server has authentication enabled](/api-keys-and-authentication#start-a-langflow-server-with-authentication-enabled).
+Each [Langflow project](/concepts-flows#projects) has its own MCP server with it's own MCP server authentication settings.
 
-When this is the case, the code template in your project's **MCP Server** tab automatically includes the `--header` and `x-api-key` arguments:
+Langflow MCP servers support these authentication types.
 
-```json
-{
-  "mcpServers": {
-    "PROJECT_NAME": {
-      "command": "uvx",
-      "args": [
-        "mcp-proxy",
-        "--headers",
-        "x-api-key",
-        "YOUR_API_KEY",
-        "http://LANGFLOW_SERVER_ADDRESS/api/v1/mcp/project/PROJECT_ID/sse"
-      ]
-    }
-  }
-}
-```
+To configure authentication for a Langflow MCP server, go to the **Projects** page in Langflow, click the **MCP Server** tab, click <Icon name="Fingerprint" aria-hidden="true"/> **Add Auth**, and then select your preferred authentication method:
+
+<Tabs groupId="auth-type">
+<TabItem value="API key" label="API key">
+
+When authenticating your MCP server with an Langflow API key, the code template in your project's **MCP Server** tab automatically includes the `--headers` and `x-api-key` arguments.
```

Comment:
```suggestion
When authenticating your MCP server with an Langflow API key, your project's MCP server **JSON** code snippets automatically include the `--headers` and `x-api-key` arguments.
```

Reply:
Idk if this means it only adds it to the JSON snippets or if it also adds it when you use auto-install.